.travis.yml

```diff
@@ -78,4 +78,4 @@ install:
 script:
   - industrial_ci/_wrap_test.sh # this script is only for internal use
   - if ! [ -z "$POST_PROCESS" ]; then $POST_PROCESS; fi
-  - if ! [ -z "$DOCKER_COMMIT" ]; then  echo "Test if the Docker image is committed or not. See the size of the Docker image is non-zero or not."; if docker image inspect $DOCKER_COMMIT --format='{{.Size}}'; then exit 0; else exit 1; fi fi
+  - if ! [ -z "$DOCKER_COMMIT" ]; then  echo "Test if the Docker image is committed or not. See the size of the Docker image is non-zero or not."; img_size=$(docker image inspect ros:melodic --format='{{.Size}}'); if [ ${img_size} -eq 0 ]; then exit 1; fi fi
```

Comment:
Why not just:
```bash
if [ -n "$DOCKER_COMMIT" ]; then docker image inspect "$DOCKER_COMMIT" --format="$DOCKER_COMMIT: {{.Size}} bytes"; fi
```
Testing for the size to be 0 does not make much sense.
But inspect will fail if the image does not exist.

Reply:
Thanks, I confirmed as below. Updated.

```
$ docker pull ros:melodic
$ DOCKER_COMMIT=ros:melodic
$ if [ -n "$DOCKER_COMMIT" ]; then docker image inspect "$DOCKER_COMMIT" --format="$DOCKER_COMMIT: {{.Size}} bytes"; fi
ros:melodic: 1270512578 bytes
$ DOCKER_COMMIT=ros:melodico
$ if [ -n "$DOCKER_COMMIT" ]; then docker image inspect "$DOCKER_COMMIT" --format="$DOCKER_COMMIT: {{.Size}} bytes"; fi

Error: No such image: ros:melodico
```